src/main/java/io/aiven/kafka/connect/gcs/GoogleCredentialsBuilder.java

```diff
@@ -44,7 +44,7 @@
      * @return a {@link GoogleCredentials} constructed based on the input.
      * @throws IOException              if some error getting the credentials happen.
      * @throws IllegalArgumentException
-     * if both {@code credentialsPath} and {@code credentialsJson} are non-{@code null}.
+     *     if both {@code credentialsPath} and {@code credentialsJson} are non-{@code null}.
```

Comment:
Accidentally?

Reply:
new checkstyle requires this indentation

Reply:
gotcha